pkg/manifests/types.go

```diff
@@ -343,6 +343,19 @@ type NodeExporterCollectorTcpStatConfig struct {
 type NodeExporterCollectorNetDevConfig struct {
 	// A Boolean flag that enables or disables the `netdev` collector.
 	Enabled bool `json:"enabled,omitempty"`
+	// A list of network devices to be excluded from collection, using regular expressions.
+	// By default, the following list is used:
+	// `veth.*`,
+	// `[a-f0-9]{15}`,
+	// `enP.*`,
+	// `ovn-k8s-mp[0-9]*`,
+	// `br-ex`,
+	// `br-int`,
+	// `br-ext`,
+	// `br[0-9]*`,
+	// `tun[0-9]*`,
+	// `cali[a-f0-9]*`
+	IgnoredNetworkDevices *[]string `json:"ignoredNetworkDevices,omitempty"`
```

Comment:
(nit) we can use a slice instead of a pointer. An empty slice means "use the default list".
 
```suggestion
	IgnoredNetworkDevices []string `json:"ignoredNetworkDevices,omitempty"`
```

Reply:
(and btw that's why I needed the ability to remove the  `--collector.netclass.ignored-devices` argument in manifest.go)